src/docs/product/data-management-settings/event-grouping/index.mdx

```diff
@@ -39,7 +39,7 @@ When you create a Sentry project, the most recent version of the grouping algori
 
 To upgrade an existing project to a new grouping algorithm version, navigate to **Settings > Projects > [project] > Issue Grouping > Upgrade Grouping**. After upgrading to a new grouping algorithm, you will very likely see new groups being created.
 
-All versions consider the `stacktrace`, `exception` and `message`.
+All versions consider the `stacktrace`, `exception` and `message`, in sequential order.
```

Comment:
Since the sequential nature of this is important, let's just be more explicit about it. 

```suggestion
All versions consider the `stack trace` first, the `exception` next, and then finally the `message`.
```